src/components/ui/command.tsx

```
@@ -0,0 +1,153 @@
+          {children}
+        </Command>
+      </DialogContent>
+    </Dialog>
+  )
+}
+
+const CommandInput = React.forwardRef<
+  React.ElementRef<typeof CommandPrimitive.Input>,
+  React.ComponentPropsWithoutRef<typeof CommandPrimitive.Input>
+>(({ className, ...props }, ref) => (
+  <div className="flex items-center border-b px-3" cmdk-input-wrapper="">
```

ESLint

[error] Unknown property 'cmdk-input-wrapper' found

(react/no-unknown-property)